src/schemas/21-4142/schema.js

```diff
@@ -0,0 +1,42 @@
+import schemaHelpers from '../../common/schema-helpers';
+
+let schema = {
+    $schema: 'http://json-schema.org/draft-04/schema#',
+    title: 'AUTHORIZATION TO DISCLOSE INFORMATION TO THE DEPARTMENT OF VETERANS AFFAIRS (21-4142)',
+    type: 'object',
+    definitions: {},
+    anyOf: [
+        {
+            "required" : ["veteranVaFileNumber"]
+        },
+        {
+            "required" : ["veteranSocialSecurityNumber"]
+        }
+    ],
+    additionalProperties: false,
+    properties: {
+        claimantEmail: {
+            type: 'string',
+            format: 'email'
+        },
+        limitedConsent: {
+            type: 'string'
+        },
+    },
+    required: ['privacyAgreementAccepted', 'claimantFullName']
+};
+
+[
+    ['fullName', 'claimantFullName'],
+    ['ssn', 'veteranSocialSecurityNumber'],
+    ['centralMailVaFile', 'veteranVaFileNumber'],
```

Comment:
we have been using `vaFileNumber` for the name of this field on other forms and generally try to keep the naming consistent unless there's a good reason not to